### DISCUSSION TITLE
element:mouseenter when selected with ui.Selection

### DISCUSSION THREAD

**Discussion by frnora** [clientIO/joint#2147] [Q&A]
### Introduction

Hi,

`element:mouseenter` (and I assume the other pointer-events as well) don't fire when an element is selected with ui.Selection.

Disabling the pointer-events on the selection doesn't work as (multi) selection cannot be dragged without them.
As I don't want to reinvent the wheel, is there a better approach than setting a mouseenter listener on the selection-box DOM element, checking the data-model attribute and then triggering the action on that model's view?

Use case: I want to show some tools on hover and hide them on mouse out, no matter whether the element is selected or not.

Thank you!

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
It will be possible in the upcoming version `v3.7`.
See this [comment](https://github.com/clientIO/joint/discussions/1669#discussioncomment-2790669) for a workaround.
